gloo/common/linux.cc

```diff
@@ -190,54 +193,65 @@ const std::string& infinibandToBusID(const std::string& name) {
 }
 
 static int getInterfaceSpeedGLinkSettings(int sock, struct ifreq* ifr) {
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,6,0)
-  constexpr auto link_mode_data_nwords = 3 * 127;
-  struct {
-    __u32 link_mode_data[link_mode_data_nwords];
```

Comment:
AFAIU this is not a "flexible array". `link_mode_data_nwords` is `constexpr`, so it is very much the same as a literal `381`.